android/app/build.gradle

```
@@ -155,8 +155,8 @@ android {
         testBuildType System.getProperty('testBuildType', 'debug')
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
         resValue "string", "app_name", project.env.get("APP_DISPLAY_NAME")
-        resValue "string", "build_config_package", "org.celo.mobile"
         resValue "string", "deep_link_url_scheme", project.env.get("DEEP_LINK_URL_SCHEME")
+        resValue "string", "build_config_package", "xyz.mobilestack.mobile"
```

What do you think of using a shorter namespace too?
```suggestion
        resValue "string", "build_config_package", "xyz.mobilestack"
```